mk-sympa-dist.pl.in

```diff
@@ -0,0 +1,134 @@
+#!--PERL--
+# -*- indent-tabs-mode: nil; -*-
+# vim:ft=perl:et:sw=4
+# $Id$
+
+# Sympa - SYsteme de Multi-Postage Automatique
+#
+# Copyright (c) 1997, 1998, 1999 Institut Pasteur & Christophe Wolfhugel
+# Copyright (c) 1997, 1998, 1999, 2000, 2001, 2002, 2003, 2004, 2005,
+# 2006, 2007, 2008, 2009, 2010, 2011 Comite Reseau des Universites
+# Copyright (c) 2011, 2012, 2013, 2014, 2015, 2016, 2017 GIP RENATER
+# Copyright 2018 The Sympa Community. See the AUTHORS.md file at the
+# top-level directory of this distribution and at
+# <https://github.com/sympa-community/sympa.git>.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use lib '--srcdir--/src/lib/';
+use strict;
+use warnings;
+use English qw(-no_match_vars);
+use Pod::Usage;
+
+use Sympa::ConfDef;
+use Sympa::Constants;
+
+my $modfail;      # any of required modules are not installed.
+
+BEGIN {
+    $modfail = !eval {
+        require Conf;
+        require Sympa::Language;
+        require Sympa::Tools::Text;
+    };
+}
+
+# Set language context if possible.
```

Comment:
Well, Sympa::Language allows to have a translated sympa.conf-dist, which make its use easier.

Reply:
However, translation catalog may not be used. Because the code doesn't refer to translation catalog bundled in the source.
By any chance, `localedir` specified in configure time could refer to translation catalog installed with previous version (if previous version had been installed). But it won't contain current translations.

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/595/commits/ec80e1b3b6bdd0c5179c94e492d244864ce163c8

Reply:
I confirmed. Thanks.